.github/CONTRIBUTING.md

```diff
@@ -20,32 +20,33 @@ Make sure you have a [GitHub Account](https://www.github.com)
 ## Project Details
 To successfully contribute, you should spend a little time familiarizing yourself with the following key topics.
 
-- [Coding & Conventions](https://github.com/phantomcyber/phantom-apps/blob/next/.github/CONVENTIONS.md) - How we expect to see code formatted and apps named
-- [Certified vs Standard App](https://github.com/phantomcyber/phantom-apps/blob/master/.github/CERTIFIED_V_UNCERTIFIED.md) definitions and differences
-- [Typical developer workflow](https://github.com/phantomcyber/phantom-apps/blob/master/.github/DEV_WORKFLOW.md) - Configuring your dev environment
-<!-- - [Testing Details](https://github.com/phantomcyber/phantom-apps/blob/next/.github/TESTING.md) - How we test apps & playbooks -->
+- [Coding & Conventions](https://github.com/splunk-soar-apps/.github/blob/main/.github/CONVENTIONS.md) - How we expect to see code formatted and apps named
```

Comment:
we seem to mix links to `next` and `main` on the `.github` repo throughout these docs (seems same issue existed before this PR). should we consolidate these to all point to the same branch?

Reply:
links to the ```.github``` repo should always reference ```main```, I'll update any links that are not